src/main/java/com/knu/buga1chuk/serialization/interfaces/Serialization.java

```diff
@@ -0,0 +1,13 @@
+package com.knu.buga1chuk.serialization.interfaces;
+
+import com.knu.buga1chuk.model.Person;
+
+import java.io.File;
+import java.util.List;
+
+public interface Serialization {
```

Comment:
Choose better naming. (It is too generic)

Reply:
Fix